awx/main/models/projects.py

```diff
@@ -284,6 +284,16 @@ class Meta:
         help_text=_('Allow changing the SCM branch or revision in a job template ' 'that uses this project.'),
     )
 
+    # credential (keys) used to validate content signature
+    signature_validation_credential = models.ForeignKey(
+        'Credential',
+        related_name='%(class)ss_signature_validation',
+        blank=True,
+        null=True,
+        default=None,
+        on_delete=models.SET_NULL,
```

Comment:
I think it would be good to add `help_text` to the field definition here to make the OPTIONS more readable.

Reply:
@djyasin can you add that?